RadarSDK/Radar.m

```diff
@@ -186,9 +186,17 @@ + (void)trackOnceWithLocation:(CLLocation *)location completionHandler:(RadarTra
 }
 
 + (void)startTrackingWithOptions:(RadarTrackingOptions *)options {
+    [RadarSettings setListenToServerTrackingOptions:NO];
     [[RadarLocationManager sharedInstance] startTrackingWithOptions:options];
 }
 
++ (void)startTrackingWithRemoteOptions {
+    [RadarSettings setListenToServerTrackingOptions:YES];
+    [[RadarAPIClient sharedInstance] getConfig:^(RadarStatus status, NSDictionary *_Nullable res) {
+        [[RadarLocationManager sharedInstance] startTrackingWithOptions:[RadarSettings trackingOptions]];
```

Comment:
on first read I expected `res` to be passed through the function as the options arg - it feels like a bit of misdirection that we're setting the storing the config while parsing the meta and then refetching the config here?